experiments/conductor/cmd/runner/utilities.go

```diff
@@ -721,6 +754,10 @@ func runGoCmds(opts *RunnerOptions, affectedPaths []string) error {
 	errStrings := []string{}
 
 	for _, path := range affectedPaths {
+		// Skip if not a Go file
```

Comment:
@jingyih - https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/4177 - This should be sufficient right ?

Reply:
So the affectedPath is a folder. It may actually contain go files?

Reply:
makes sense i will approve your PR. Please review rest of this PR